### DISCUSSION TITLE
FSNotes 5 for iOS

### DISCUSSION THREAD

**Discussion by glushchenko** [glushchenko/fsnotes#1293] [Announcements]
Hi, guys!

I am hard working on FSNotes 5 for iOS. First beta coming before february 20 or even early.

This will be the biggest update ever for mobile fsnotes. Many bugs have been fixed, the application has become even more native, new cool features have been added.

You asked me about icons, it's time to think about it, but personally im not really good in icons design.

If someone wants to draw an icon for an application, especially in a dark style, it's time to do it and help the project. Just let me know.

**Comment by gingerbeardman** [comment]
Current v4 icon

![Untitled](https://user-images.githubusercontent.com/49612/153581895-ef7c8dd1-c692-4e52-ba42-0a0642ff8ff9.png)

Recent icon design from @nth-chile https://github.com/glushchenko/fsnotes/issues/1264

> 
![](https://user-images.githubusercontent.com/22358233/146282005-3f1ced6b-d2fc-4759-ba55-5012c8e99076.png)

Perhaps good to start discussion from the icon above.

What are the main features of v5? Anything that might lend itself to the icon design?

I think if we could provide an icon for dark mode too https://github.com/glushchenko/fsnotes/issues/1049

**Comment by gingerbeardman** [comment]
Some ideas for me. 


**None of these are meant to be the icon! Just early ideas.**

<img width="196" alt="Screen shot 2022-02-11 at 12 30 16" src="https://user-images.githubusercontent.com/49612/153591797-a145087d-e566-43b3-81cc-799dd323e8a5.png">

**Comment by nth-chile** [comment]
Excited about the updates! Those are cool @gingerbeardman. Here's another idea borrowing from the existing logo 

<img width="182" alt="Screen Shot 2022-02-11 at 11 13 10 AM" src="https://user-images.githubusercontent.com/22358233/153627377-7f1cc97f-d0ec-4856-8d91-6ac294e73edc.png">

**Comment by gingerbeardman** [comment]
What are the required items on the icon?

- something that indicates "note"
- lightning bolt?
- "FS"?
- the number 5?

**Reply by nth-chile** [reply]
I would +1 "note", however I notice that not a whole lot of notes apps and code editors really do that. Apple's text editors tend to look like notes, but Bear,  Notion, VSCode, Sublime Text really don't.

We could also riff off of folders/files icons, since FS stands for filesystem. But also, Finder is a smiley face, so idk if there are rules :)

**Comment by glushchenko** [comment]
I think the icon can be anything. The main one will remain old, but it would be nice if there was a choice in the settings.

**Comment by nth-chile** [comment]
Could also have a mascot, like the Notepad++ chameleon

**Comment by gingerbeardman** [comment]
I think the right way to go about this and get a great result would be to approach a pro icon designer, maybe @glushchenko could check out [Dribbble](https://dribbble.com/search/macos-icon) to see if there are any artists that you like? Of course it would cost a little, but I think it's worthwhile.

For example Noteship has a great icon.
https://dribbble.com/shots/16513644-Noteship-macOS-App-Icon

**Reply by nth-chile** [reply]
https://dribbble.com/raphaellopesph

**Reply by gingerbeardman** [reply]
Not dribble but https://motorcycl3.com/beos/
